front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/Itinerary.tsx

```diff
@@ -23,12 +15,20 @@ import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
 import { GoArrowSwitch, GoPlus, GoRocket, GoTrash } from 'react-icons/go';
 import Tipped from 'applications/editor/components/Tipped';
 import TypeAndPath from 'common/Pathfinding/TypeAndPath';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Reply:
Done